src/scripts/api.ts

```diff
@@ -263,6 +267,13 @@ export class ComfyApi extends EventTarget {
   user: string
   socket: WebSocket | null = null
 
+  /**
+   * Cache Firebase auth store composable function.
+   */
+  private authStoreComposable:
+    | (() => ReturnType<typeof useFirebaseAuthStore>)
+    | null = null
```

Comment:
try to simplify the type to just `typeof useFirebaseAuthStore | undefined`. use optional syntax.